pkg/registry/controller/rest.go

```diff
@@ -45,23 +83,24 @@ func NewREST(registry Registry, podLister PodLister) *REST {
 	return &REST{
 		registry:  registry,
 		podLister: podLister,
+		strategy:  Strategy,
 	}
 }
 
-// Create registers the given ReplicationController.
+// Create registers the given ReplicationController with the controller manager.
```

Comment:
Nit: if we want to be pedantic we could say it adds it to the system, which eventually has the effect that a controller manager will begin acting in its behalf.